dags/pytorch_xla/configs/pytorchxla_torchbench_config.py

```diff
@@ -47,18 +103,16 @@ def model_install_cmds(output_file=None) -> str:
       "sudo apt-get install libgl1 -y",
       "pip3 install --user numpy pandas",
       (
-          "pip3 install --user --pre torch torchvision torchaudio --index-url"
-          " https://download.pytorch.org/whl/nightly/cpu"
+          f"pip3 install --user --pre {version_mapping.TORCH.value} {version_mapping.TORCHVISION.value} {version_mapping.TORCHAUDIO.value} --index-url {version_mapping.TORCH_INDEX_CPU_URL.value}"
```

Comment:
This is only for nightly? Or it could also used for installing stable torch wheels?

Reply:
This can be adapted for both nightly and other torch wheels. However, we have to find the matching version of torchvision and torchaudio.

Reply:
So it could be used for installing stable torch wheels.

Reply:
Right!